Jasmine/Common/Models/TextGrid.swift

```diff
@@ -87,11 +87,13 @@ struct TextGrid {
     ///
     /// - Parameter coordinates: Array of Coordinate to get the text at
     /// - Returns: Array of String correspond to the texts at `coordinates`
-    ///            Returns nil if there is no text at any of the coordinates
+    ///            Returns nil if any of the coordinates is out of bounds or has no text
     func getTexts(at coordinates: [Coordinate]) -> [String]? {
         var texts: [String] = []
         for coordinate in coordinates {
-            guard let text = self[coordinate] else {
+
+            guard 0..<numRows ~= coordinate.row && 0..<numColumns ~= coordinate.col,
```

Comment:
You don't want to just crash when it's out of bounds? Makes more sense to do so, no, since it's a programmer error? (Think of it like an assert)

Anyways there's the `isInBounds` function.